sdk/src/test/java/com/google/cloud/dataflow/sdk/util/common/CounterSetTest.java

```diff
@@ -222,4 +226,127 @@ public void testAddCounterMutatorAddMultipleCounters() {
     assertThat(set, containsInAnyOrder(c1, c2));
   }
 
+  @Test
+  public void testNullCounterBeforeAdd() {
+    final String counterName = "counter-name";
+    CounterSet counters = new CounterSet();
```

Comment:
Can make this a final field in the test class, so it doesn't need to be created in each test.